ftw/blueprints/sections/multilingual.py

```diff
@@ -1,11 +1,11 @@
 from collective.transmogrifier.interfaces import ISection
 from collective.transmogrifier.interfaces import ISectionBlueprint
 from collective.transmogrifier.utils import defaultMatcher
-from plone.multilingual.interfaces import ILanguage
-from plone.multilingual.interfaces import IMutableTG
-from plone.multilingual.interfaces import ITranslatable
-from plone.multilingual.interfaces import ITranslationManager
+from plone.app.multilingual.interfaces import IMutableTG
```

Comment:
Those imports do not work on Plone 4:
```python
ImportError: cannot import name IMutableTG
```

I think we need to do conditional imports in order to support both Plone versions.
We still have a lot of Plone 4's running and are not yet ready to drop Plone 4 support in `ftw.blueprints`.

Could you change the imports to support both versions?